include/mlir/Analysis/Dominance.h

```diff
@@ -90,6 +90,12 @@ class DominanceInfo : public detail::DominanceInfoBase</*IsPostDom=*/false> {
   bool properlyDominates(Block *a, Block *b) {
     return super::properlyDominates(a, b);
   }
+
+  /// Return the dominanceInfos from the Region containing block A.
```

Comment:
dominanceInfos doesn't make sense here, can this change to 'dominance node'. This should also change the llvm::DomTreeNodeBase<Block> -> DominanceInfoNode

Reply:
change pushed